src/test/java/nextstep/subway/unit/LineServiceTest.java

```diff
@@ -156,4 +157,72 @@ void addSectionContainsNoStation() {
                 lineService.addSection(신분당선.getId(), 새로운_요청)
         ).isInstanceOf(SectionContainsAnyStationException.class);
     }
+
+    @DisplayName("하행 종점역 삭제")
+    @Test
+    void removeDownStation() {
+        //when
+        //강남역과 분당역 사이에 정자역 추가
+        Station 정자역 = stationRepository.save(new Station("정자역"));
+        lineService.addSection(신분당선.getId(),
+                new SectionRequest(강남역.getId(), 정자역.getId(), 4));
+        //Then
+        //하행 종점역을 삭제
+        lineService.deleteSection(신분당선.getId(), 분당역.getId());
+        assertThat(신분당선.getStations()).containsExactly(강남역,정자역);
```

Comment:
다음과 같이 given-when-then 이 되야할 것 같아요 😄 
```suggestion
    @DisplayName("하행 종점역 삭제")
    @Test
    void removeDownStation() {
        //given
        //강남역과 분당역 사이에 정자역 추가
        Station 정자역 = stationRepository.save(new Station("정자역"));
        lineService.addSection(신분당선.getId(),
                new SectionRequest(강남역.getId(), 정자역.getId(), 4));
        //when
        //하행 종점역을 삭제
        lineService.deleteSection(신분당선.getId(), 분당역.getId());
        
        // then
        assertThat(신분당선.getStations()).containsExactly(강남역,정자역);
```